.github/workflows/main.yml

```diff
@@ -1,18 +1,24 @@
-name: Ruby
+name: Ruby Tests
 
-on: [push,pull_request]
+on: [push, pull_request]
 
 jobs:
-  build:
+  test:
+    name: Ruby ${{ matrix.ruby-version }}
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: ['3.0.1', '3.1.1', '3.2.1', '3.3.1', '3.4.1']
```

Comment:
I think we can remove `3.0.1` from here. According to https://www.ruby-lang.org/en/downloads/branches/, 3.0 is already EOL.

Reply:
Done in [77f4d9e](https://github.com/degica/omni_exchange/pull/9/commits/77f4d9e83227cbf4fa48a5db8980a63010309065).